pkg/proxy/accesslog/record.go

```diff
@@ -195,3 +195,22 @@ type LogRecordHTTP struct {
 	// Header is the HTTP header in use
 	Header http.Header
 }
+
+
+
+// LogRecordKafka contains the kafka specific portion of a log record
+// Reference: https://kafka.apache.org/protocol#protocol_api_keys
+type LogRecordKafka struct {
+	// Version of the kafka api used
+	version int
+
+	// Kafka ApiKey -> produce/fetch/createTopic/deleteTopic
```

Comment:
You should report all types of messages. Don't limit to only a subset.

Reply:
The comment is just an example. Will remove it.